lib/cli/utils.js

```diff
@@ -88,7 +88,8 @@ utils.iterativeCompletion = (inputs, depth, completer) => {
   // they'll be not valid (those are completions for previous inputs)
   if (newDepth <= inputs.length - 1) completions = [];
   if (!completions[0] ||
-      completions.length === 1 && completions[0] === inputs[depth]) {
+    completions.length === 1 && completions[0] === inputs[depth]
```

Comment:
It's good but the name is slightly misleading as this also checks that this array is empty or first element is false (empty string in this case).